app/views/scripts.scala.html

```diff
@@ -24,4 +24,6 @@
 
     @hmrcScripts()
 
- <script @{CSPNonce.attr} src="@routes.Assets.versioned("javascripts/self-service-time-to-pay.js")"></script>
\ No newline at end of file
+ <script @{CSPNonce.attr} src="@routes.Assets.versioned("javascripts/self-service-time-to-pay.js")"></script>
+
+ <script @{CSPNonce.attr} src="https://cdn.optimizely.com/js/11766650496.js"></script>
```

Comment:
if we're got the ID in config we should probably be reading it from there to stick in this URL so that this gets automatically updated if the ID changes